src/applications/debt-letters/const/diary-codes/index.js

```diff
@@ -436,104 +334,72 @@ export const renderAdditionalInfo = (diaryCode, dateOfLetter) => {
     case '425':
     case '627':
       return (
-        <div>
-          <p>
-            <strong>Status: </strong>
-            We're updating your account.
-          </p>
-          <p>
-            <strong>Next step: </strong>
-            Please check back in
-            <strong> 30 days </strong>
-            for updates. If your account shows the same information then, call
-            us at 800-827-0648 (or 1-612-713-6415 from overseas). We’re here
-            Monday through Friday, 7:30 a.m. to 7:00 p.m. ET.
-          </p>
-        </div>
+        <p>
+          <strong>Next step: </strong>
+          Please check back in <strong> 30 days </strong> for updates. If your
+          account shows the same information then, call us at 800-827-0648 (or
+          1-612-713-6415 from overseas). We’re here Monday through Friday, 7:30
+          a.m. to 7:00 p.m. ET.
```

Comment:
`<ContactDMC />`? This reminds me of the claim status tool code

<img alt="ghe-emoji" title=":_tuzki_dizzy:" src="https://cloud.githubusercontent.com/assets/136959/13203535/61480e08-d880-11e5-9ea8-7fc03c7d7b56.gif">